WebCoffeeMachine/coffeemachine/templates/coffeemachine/detail.html

```diff
@@ -36,13 +36,17 @@ <h1>Your Selection</h1>
         <div id="overlay">
             <h1 id="overlay-text">Please make the payment</h1>
             <img id="overlay-img" src="" alt="alt-image">
+            <h2 id="overlay-homeBtn"><a href={% url 'coffeemachine:home' %}>Home Page</a></h2>
         </div>
         <script>
             document.getElementById("overlay-img").src = "{% get_static_prefix %}coffeemachine/images/cashless_payment-transparent.png"; 
             let element = document.getElementById("price-div")
             let data = element.dataset
             let unit_price = parseFloat(data.price)
             let clicks = 1;
+            let count
```

Comment:
In JavaScript, semicolons are used to end a statement. While they are not always required, it is considered good practice to include them to avoid potential errors.